tests/base.py

```diff
@@ -223,6 +230,14 @@ def setUp(self, mock_verify_calendar_id):
                  location_id=1
             )
             child_node.save()
+            child_node_2 = OfficeStructure(
+                id='C56A4180-65AA-42EC-A945-5FD21DEC0528',
+                name='Kampala',
+                tag='Forth Floor',
+                parent_id='C56A4180-65AA-42EC-A945-5FD21DEC0527',
+                location_id=1
+            )
+            child_node_2.save()
```

Comment:
I don't think you need to add the `parent` and `child` nodes. You can use the existing ones for your test.

Reply:
Done @rajeman